deepset_cloud_sdk/_s3/upload.py

```diff
@@ -162,9 +163,14 @@ async def upload_from_string(
         try:
             await self._upload_file_with_retries(file_name, upload_session, content, client_session)
             return S3UploadResult(file_name=file_name, success=True)
-        except:  # pylint: disable=bare-except
-            logger.warn("Could not upload a file to S3", file_name=file_name, session_id=upload_session.session_id)
-            return S3UploadResult(file_name=file_name, success=False)
+        except Exception as exception:  # pylint: disable=bare-except, disable=broad-exception-caught
+            logger.warn(
+                "Could not upload a file to S3",
```

Comment:
should we rather say "Could not upload a file to deepset Cloud" since users don't know that we're using S3 buckets under the hood?